plutus-tx/compiler/Language/PlutusTx/Data.hs

```diff
@@ -27,35 +28,53 @@ import           GHC.Generics
 data Data =
       Constr Integer [Data]
     | Map [(Data, Data)]
+    | Seq [Data]
```

Comment:
Any particular reason it's called Seq and not Map?

This prompts me to wonder whether we need to care about the semantics of `Data`.  One interpretation of Seq is linked lists, but another could be random access arrays/vectors with constant access time.  I suppose it's up to the scripting language how it deals with these things.

Reply:
Not particularly. I'll just call it `List`.